framework/.changeset/v0.7.8.md

```diff
@@ -0,0 +1 @@
+- Solidity storage layout helper for anvil_setStorageAt + tests + guide
```

Comment:
please bump to 0.7.9, because 0.7.8 will be released very shortly (or would have already been released by the time you read it)